Atomic/backendutils.py

```diff
@@ -62,7 +62,7 @@ def get_backend_and_image(self, img, str_preferred_backend=None):
         raise ValueError("Found {} in multiple storage backends: {}".
                          format(img, ', '.join([x.backend for x in img_in_backends])))
 
-    def get_backend_for_container(self, container, str_preferred_backend=None):
+    def get_backend_and_container(self, container, str_preferred_backend=None):
```

Comment:
Confusing name of method.  You are passing in "container" and returning con_obj.  Either
container should be renamed to container_id, or function should be get_backend_and_container_object